examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    hierarchical_df["top_level"].replace({"hol": "Hol", "vfr": "VFR", "bus": "Bus", "oth": "Oth"}, inplace=True)
</pre><p>Do we really need to to it?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>This is added only for convenience. No confusion , easy merging/comparison with the initial dataset.</p>

Reply:
<p>Don't get what do we need it for</p>

Reply:
<p>Only for reasons of name consistency with the original dataset.</p>